search/engine.go

```diff
@@ -499,9 +503,16 @@ func (e *ESEngine) timeTrack(start time.Time, operation string) {
 func (e *ESEngine) DoSearch(ctx context.Context, query Query, sortBy string, from int, size int, preference string) (*QueryResult, error) {
 	defer e.timeTrack(time.Now(), "DoSearch")
 
-	if err := e.AddIntents(&query, preference); err != nil {
-		return nil, errors.Wrap(err, "ESEngine.DoSearch - Error adding intents.")
-	}
+	done := make(chan bool)
+	go func() {
+		intents, err := e.AddIntents(&query, preference, size)
```

Comment:
Why size? Why not constant of 10?
I think we don't want this number to change. We just want it to be lower, i.e., not 50.